team-apm/apm
# feat: UI improvement

## Description

Improved the design of the main tab and the plugins & scripts tab. Especially, usability in small window sizes is improved.

## Related Issue





## Motivation and Context


## How Has This Been Tested?



- Resize the window to make sure the design is intact.
- Check the visibility in the dark theme.

## Screenshots (if appropriate):

## Types of changes


## Checklist:
